test/python/visualization/test_visualization.py

```diff
@@ -209,6 +210,102 @@ def test_get_layered_instructions_remove_idle_wires(self):
         self.assertEqual(exp,
                          [[(op.name, op.qargs, op.cargs) for op in ops] for ops in layered_ops])
 
+    def test_get_layered_instructions_left_right_justification(self):
+        """ _get_layered_instructions illustrate left/right justification since #2802 """
```

Comment:
Could you add the circuit drawing here? Just so that it is very clear what this test is testing

Reply:
hmm, I'm trying this and two problems:
1. The lines will be too long for pycodestyle
2. I can't get the assertEqual to work. I've pasted the output every which way into the test and it doesn't pass the diff. Me confused.

Reply:
I've been trying stuff like this:
```

        drawing = """        ┌────────────┐┌───┐┌────────────┐              ┌─┐┌────────────┐┌───┐┌────────────┐
q_0: |0>┤ U2(0,pi/1) ├┤ X ├┤ U2(0,pi/1) ├──────────────┤M├┤ U2(0,pi/1) ├┤ X ├┤ U2(0,pi/1) ├
        ├────────────┤└─┬─┘├────────────┤┌────────────┐└╥┘└────────────┘└─┬─┘├────────────┤
q_1: |0>┤ U2(0,pi/1) ├──■──┤ U2(0,pi/1) ├┤ U2(0,pi/1) ├─╫─────────────────■──┤ U2(0,pi/1) ├
        └────────────┘     └────────────┘└────────────┘ ║                    └────────────┘
q_2: |0>────────────────────────────────────────────────╫──────────────────────────────────
                                                        ║                                  
q_3: |0>────────────────────────────────────────────────╫──────────────────────────────────
                                                        ║                                  
q_4: |0>────────────────────────────────────────────────╫──────────────────────────────────
                                                        ║                                  
c1_0: 0 ════════════════════════════════════════════════╩══════════════════════════════════
"""
        self.maxDiff = None
        self.assertEqual(str(qc.draw(justify='left', line_length=200)), drawing)
```
which doesn't diff correctly.

Reply:
I just meant adding them in the comment so that it was clear what the test was testing, not updating the test

Reply:
Done, thanks.